buildspec.yml

```diff
@@ -0,0 +1,75 @@
+version: 0.2
+
+env:
+  variables:
+    FRAMEWORK_VERSION: "1.3.0"
+    ACCOUNT: "142577830533"
+    AWS_DEFAULT_REGION: "us-west-2"
+    ECR_REPO: "sagemaker-framework-pull-request-integ"
+    PREPROD_IMAGE: "142577830533.dkr.ecr.us-west-2.amazonaws.com/sagemaker-framework-pull-request-integ"
+    CPU_FRAMEWORK_BINARY: "https://files.pythonhosted.org/packages/71/64/49c5125befd5e0f0e17f115d55cb78080adacbead9d19f253afd0157656a/mxnet-1.3.0.post0-py2.py3-none-manylinux1_x86_64.whl"
+    CPU_PY_VERSION: "2"
+    CPU_INSTANCE_TYPE: "ml.c4.xlarge"
+
+
+phases:
+  pre_build:
+    commands:
+      - start-dockerd
+
+
+  build:
+    commands:
+      # install
+      - pip3 install -U .[test]
+
+      # run unit tests
+      - pytest test/unit
+
+      # build cpu image
+      - |
+        build_dir="docker/$FRAMEWORK_VERSION/final"
+        cpu_dockerfile="Dockerfile.cpu"
+
+        # Download framework binary
+        cpu_fw_binary=$(basename $CPU_FRAMEWORK_BINARY)
+        wget -O $build_dir/$cpu_fw_binary $CPU_FRAMEWORK_BINARY
+
+        # Create tensorflow-container pip archive
+        python3 setup.py sdist
+        # tar_name will be something like: sagemaker_mxnet_container-1.0.tar.gz
+        tar_name=$(ls dist)
+        cp dist/$tar_name $build_dir
+
+        build_id="$(echo $CODEBUILD_BUILD_ID | sed -e 's/:/-/g')"
+        CPU_TAG="$FRAMEWORK_VERSION-cpu-py$CPU_PY_VERSION-$build_id"
```

Comment:
we could just use the build id as the tag, since these are pretty transient images

Reply:
If we use same repo, we should have at least framework_version here with build id.

Reply:
right. actually, if we are sharing the repo between projects, we should probably add framework name.

Reply:
I feel fine for not having framework name. Because build id is here, and build id will contain the build name which contains the framework name.

Reply:
ok